build.gradle

```diff
@@ -99,7 +99,7 @@ run {
 }
 
 checkstyle {
-    toolVersion = '9.3'
+    toolVersion = '10.10.0'
```

Comment:
I think it would be better if we do the checkstyle upgrade in a different PR, because there may be some changes that break backward compatibility.

Reply:
I see, sure thing, I will revert back to the previous commit before the introduction of the new checkstyle tool!